### PR TITLE
IOS-11089 Migrate all UIAlertViews to UIAlertControllers

### PR DESCRIPTION
**Pull request by jdizoglio**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jdizoglio has signed the CLA. Thanks for the pull request!

File: BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXCreateFolderViewController.m [hunk 1]
```diff
@@ -191,12 +191,17 @@ - (void)didFailToCreateFolderWithError:(NSError *)error
...
-                                                       delegate:nil
-                                              cancelButtonTitle:NSLocalizedString(@"Dismiss", @"Label: Allow the user to dismiss the current view or interface, often used on buttons to dismiss alerts")
-                                              otherButtonTitles:nil];
-    [alertView show];
+    UIAlertController *alertController = [UIAlertController alertControllerWithTitle:title
+                                                                             message:message
+                                                                      preferredStyle:UIAlertControllerStyleAlert];
+    
+    UIAlertAction *dismissAction = [UIAlertAction actionWithTitle:NSLocalizedString(@"Dismiss", @"Label: Allow the user to dismiss the current view or interface, often used on buttons to dismiss alerts")
+                                                       style:UIAlertActionStyleDefault
```
Comment (boxtcarpel) [inline BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXCreateFolderViewController.m]: UIAlertActionStyleCancel ?

Comment (boxtcarpel) [inline BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXCreateFolderViewController.m]: Or did you check it out and it looked weird?

File: BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXFolderViewController.m [hunk 2]
```diff
@@ -236,58 +236,77 @@ - (UITableViewCellEditingStyle)tableView:(UITableView *)tableView editingStyleFo
...
+    
+    BOXErrorBlock errorBlock = ^(NSError *error){
+        dispatch_async(dispatch_get_main_queue(), ^{
+            [MBProgressHUD hideAllHUDsForView:self.view animated:YES];
+            if (error) {
+                UIAlertController *alertController = [UIAlertController alertControllerWithTitle:NSLocalizedString(@"Unable to Delete", @"Label: title for item deletion failure alert")
+                                                                                         message:NSLocalizedString(@"Unable to delete this item.", @"Label: alert message for item deletion failure")
+                                                                                  preferredStyle:UIAlertControllerStyleAlert];
+                
+                UIAlertAction *okAction = [UIAlertAction actionWithTitle:NSLocalizedString(@"Ok", @"Label: Ok action ")
```
Comment (boxtcarpel) [inline BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXFolderViewController.m]: I believe we use "Dismiss" rather than "Ok" (or "OK")

**Comment by boxtcarpel** [comment]
A couple of comments, otherwise LGTM :keycap_ten:

File: BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXCreateFolderViewController.m (hunk 1 above)
Comment (jdizoglio) [inline BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXCreateFolderViewController.m]: changing, this was an oversight on my part. Looks good.

File: BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXFolderViewController.m (hunk 2 above)
Comment (jdizoglio) [inline BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXFolderViewController.m]: done

**Pull request merged**